index.html

```diff
@@ -1,9 +1,30 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>The Best Recipes!</title>
+    <title>My Favorite Recipes!</title>
+    <link rel="stylesheet" href="styles.css" />
   </head>
   <body>
-    <h1>Recipes</h1>
+    <h1>Favorite Recipes</h1>
+    <div>
```

Comment:
An alternative to div, when we want to divided our pages into sections would be the `<section>` tag